### PR TITLE
Use accounting contact

### PR DESCRIPTION
**Pull request by corvus-ch**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc [hunk 1]
```diff
@@ -36,7 +36,9 @@ If the pod has been deleted, you can also use https://logging.apps.cloudscale-lp
 Use the content of the field `source` as a starter.
 Use https://portal.appuio.cloud to look up the organization, and then use https://control.vshn.net to identify a matching customer.
 Once a matching customer was found, look it up in https://erp.vshn.net.
-Copy the ID from the URL.
+Search for the customer's accounting contact.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc:39]: How is a customer's accounting contact marked in Odoo?

**Review by simu — APPROVED** [review]
LGTM, might be nice to have a bit more detail on how to idenitfy an accounting contact.

File: docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc (hunk 1 above)
Comment (corvus-ch) [reply docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc:39]: It is named `<customer name>, Accounting`.

Comment (corvus-ch) [inline docs/modules/ROOT/pages/runbooks/APPUiOCloudReportingMissingData.adoc:39]:
```suggestion
Search for the customer's accounting contact.
It is named `<customer name>, Accounting`.
```

**Pull request merged**